checkfiles.py

```diff
@@ -1002,11 +1002,16 @@ def fetch_files(session, url, search_query, out, include_unexpired_upload=False,
         }
         errors = job['errors']
         item_url = urljoin(url, job['@id'])
-
+        fileObject = session.get(item_url)
         r = session.get(item_url + '@@upload?datastore=database')
         if r.ok:
             upload_credentials = r.json()['@graph'][0]['upload_credentials']
-            job['upload_url'] = upload_credentials['upload_url']
+            try: 
+                if fileObject.json()['s3_uri']:
+                    job['s3_uri'] = fileObject.json()['s3_uri']
+            except:
```

Comment:
Should be explicit about the type of exception:
```suggestion
            except KeyError:
```